src/styl/responsive/mobile.styl

```diff
@@ -1,2 +1,34 @@
+$border-style = 1px solid $color-black
+
 .border-top
-  border-top: 1px solid $color-black
+  border-top: $border-style
+
+#postit
+  position: absolute
+  z-index: -1
+  width: 150px
+  transform: rotate(-10deg)
+
+#postit-dot
+  position: absolute
+  top: -10px
+  left: 45%
+  background-color: $color-black
+
+// ******************************************************
+
+// HEADER
+
+#main-nav
+  ul.grid-row
+    justify-content: space-between
```

Comment:
if this is just targeting one list I think the ID selector should be on the `ul`, otherwise quite an inefficient selector I think